plex_set_tracks.py

```diff
@@ -87,11 +87,9 @@ def getIndexFromStream(self, givenStream):
         """ Return 1-index of given :class:`~plexapi.media.AudioStream` or 
             :class:`~plexapi.media.SubtitleStream`. """
         streams = self.allStreams()
-        count = 1
-        for stream in streams:
+        for i, stream in enumerate(streams):
```

Comment:
Very nitpicky, but it might make it a tad more readable to use `for i, stream in enumerate(streams, 1)` and then just return i.  That way it's clear that the loop is 1-indexed.